src/components/pages/MainQuizPage/MainQuizPage.tsx

```diff
@@ -0,0 +1,28 @@
+import { QuestionContainer } from "./QuestionContainer/QuestionContainer";
+import { EndQuizButton } from "./EndQuizButton/EndQuizButton";
+import { ProgressBarContainer } from "./ProgressBarContainer/ProgressBarContainer";
+import { TimerContainer } from "./TimerContainer/TimerContainer";
+
+const MainQuizPage = () => {
+    return (
+        <div className="main-quiz-page-container">
+            <QuestionContainer
+                question={{
+                    question: 'What is the rigth answer?',
+                    choices: ['a', 'b', 'c', 'd'],
+                    correctAnswer: 'a',
+                    type: 'group'
+                }}
```

Comment:
[nitpicking]
could you please extract this question value into a new constant out of the component?